spec/classes/config/global/feature_config_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+describe 'rundeck' do
+  context 'supported operating systems' do
+    %w(Debian RedHat).each do |osfamily|
+      feature_config_hash = {
+        'feature.incubator.parallelWorkflowStrategy' => true
+      }
+
+      let(:facts) do
+        {
+          osfamily: osfamily,
+          fqdn: 'test.domain.com',
+          serialnumber: 0,
+          rundeck_version: '',
+          puppetversion: Puppet.version
+        }
+      end
+
+      let(:params) do
+        {
+          feature_config: feature_config_hash
+        }
+      end
+
+      # content and meta data for passwords
+      it 'generates feature_config content for rundeck-config.groovy' do
+        content = catalogue.resource('file', '/etc/rundeck/rundeck-config.groovy')[:content]
+        expect(content).to include('feature.incubator.parallelWorkflowStrategy = "true"')
```

Comment:
the generated groovy file doesn't contain the line you're looking for.
https://travis-ci.org/voxpupuli/puppet-rundeck/jobs/136383375#L1229